### PR TITLE
[Refactor](Tvf) delete some unused code of tvf and add doc for `queries` tvf

### PR DESCRIPTION
**Pull request by BePPPower**
## Proposed changes

Issue Number: close #xxx

1. delete some unused code of tvf
2. add doc for `queries` tvf: #25051 

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BePPPower** [comment]
run buildall

**Review by DongLiang-0 — APPROVED** [review]

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

File: docs/en/docs/sql-manual/sql-functions/table-functions/queries.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
```
Comment (morningman) [inline docs/en/docs/sql-manual/sql-functions/table-functions/queries.md:1]: Add this to sidebar

**Comment by BePPPower** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**